### ISSUE TITLE
GhostWriter on Windows 7 - 32 bit ?

### ISSUE THREAD

**Issue by rshalaev** [wereturtle/ghostwriter#711]
I'm trying to get GhostWriter runnign on Windows 7 - 32 bit, but was only able to find Windows 10 64bit binaries. 

    * Are there 32 bit binaries available?
    * If binaries are not available, would it be possible to compile GhostWriter 32 bit version for Windows 7? - I saw instructions for compiling it using Visual Studio, wanted to check for any known issues (esp for 32bit/Windows7)
    * Was 32bit / Windows 7 supported in earlier versions?

Help much appreciated!
Thank you

**Comment by cdwijs** [comment]
I tried to compile ghostwriter on Windows 10 using Qt 5.14.2 and the mingw 32 bit compiler. I got the following error message after opening ghostwriter.pro and pressing configure:
:-1: error: Project ERROR: Unknown module(s) in QT: webenginewidgets webengine

I added Qt Webengine, but the above error remained.

This website mentions "WebEngine is not available with MinGW under Windows. Working with Visual Studio"
https://github.com/appveyor/ci/issues/1899

I'm not able to generate a 32 bit windows build at the moment. Any idea's?
